src/views/Upload.vue

```diff
@@ -39,27 +65,131 @@ import UploadTab from '@/components/Upload/UploadTab.vue';
 import UploadFile from '@/components/Upload/UploadFile.vue';
 import UploadText from '@/components/Upload/UploadText.vue';
 
+import axios from 'axios';
+
 export default {
 	components: {
 		Navigation,
 		UploadTab,
 		UploadFile,
-		UploadText,
+		UploadText,	
 	},
 	data: function () {
 		return {
 			selectedTab: '',
 			tabs: ['Nahrať súbory', 'Vložiť text'],
+			files: [],
+			text: '',
+			uploadPercentage: 0,
+			submitted: false,	
+			alert: {
+				message: 'Nezadali ste súbor ani text, ktorý chcete skontrolovať !',
+				type: 'error',
+				duration: 4000,
+			}
 		};
 	},
+	methods: {
+		submitText(){
+			//console.log('text');
+			let formData = new FormData();
+
+			formData.append('file', this.text);
+			formData.append('title','text...');
```

Comment:
This part I used for sending text. Uploaded files should have file field and title field therefore I send text in file field and some description as title. But I think this is backend issue and first Vlado should set up text upload on backend. Then there will be something like that:
`
formData.append('text', this.text);
`

Reply:
I think this is totally unrelated to frontend -> you're posting different headers, therefore the backend should differentiate between content type `html-www-form-encoded` and `plain/text` and should not require the `title` in case of the latter.  

Please at least commentary with `// TODO (dmensa): and explanation